eessi_bot_job_manager.py

```diff
@@ -113,8 +113,15 @@ def get_current_jobs(self):
             squeue_cmd,
             "get_current_jobs(): squeue command",
             log_file=self.logfile,
+            raise_on_error=False,
         )
 
+        if squeue_exitcode != 0:
+            current_jobs = {}
```

Comment:
```suggestion
            current_jobs = {'slurm manager failed'} 
```

Maybe we can do this and than we can first check for `determine_running_jobs`, `determine_new_jobs`, `determine_finished_jobs` if the `current_jobs` holds running jobs or if has nothing because the slurm was not availabel.